scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -277,6 +277,26 @@ test-linux-stable-extra:
     - time cargo test --doc --workspace --locked --release --verbose --features runtime-benchmarks --manifest-path ./bin/node/cli/Cargo.toml
     - rusty-cachier cache upload
 
+# This job runs all benchmarks defined in the `/bin/node/runtime` once to check that there are no errors.
+short-benchmark:
```

Comment:
"short" here sounds a bit weird? Maybe `node-benchmarks`, `partial-benchmarks` or `quick-benchmarks`?

Reply:
I borrowed the name from the Polkadot side :D